### PR TITLE
Local database

### PR DESCRIPTION
**Pull request by yahyaHB**
Created local database to test and to deal with easier.

### CODE REVIEW

**Comment by astroash** [comment]
☝️ Can you please add a description to the body explaining what you are doing in this PR.

**Comment by yahyaHB** [comment]
@astroash is it clearer now?

File: src/server/models/database/dbConnection.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
 require('env2')('./config.env');
+let data_url;
 
 if (!process.env.DATABASE_URL) {
   throw new Error('DB url not found ');
+}
+if (process.env.NODE_ENV === 'test') {
+  data_url = process.env.DATABASE_URL_TEST;
 } else {
-  module.exports = new Pool({ connectionString: process.env.DATABASE_URL, ssl: true });
```
Comment (walaamedhat) [inline src/server/models/database/dbConnection.js]: why you change this line ??

Comment (yahyaHB) [reply src/server/models/database/dbConnection.js]: to deal with the local db first

File: src/client/actions/studentProfile.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import {
```
Comment (astroash) [inline src/client/actions/studentProfile.js]: This should not be coming up as a new file. Can you merge this branch with master?

**Pull request merged**